controller/lib/core/blower_pid.h

```diff
@@ -27,9 +27,7 @@ void blower_pid_init();
 // the blower to attempt to achieve the given setpoint pressure and
 // opening/closing the expire solenoid.
 //
-// Puts readings from e.g. the pressure sensor into `readings`, and puts the
-// current fan power (range [0-1]) into `fan_power`.
-void blower_pid_execute(const BlowerSystemState &desired_state,
-                        SensorReadings *readings, float *fan_power);
+float blower_pid_execute(Time now, const BlowerSystemState &desired_state,
+                         const SensorReadings &sensor_readings);
```

Comment:
It's not clear to me that the function `blower_pid_execute` should return the fan PWM setting in the range [0, 1].  Can we clarify the name and the function comment?

Reply:
yes